$NetBSD$

Pretend NetBSD is Linux for the choice of components to build.
XXX Maybe do this for *BSD, not just NetBSD.

--- tests/util/CMakeLists.txt.orig	2025-03-08 15:31:15.000000000 +0000
+++ tests/util/CMakeLists.txt
@@ -129,7 +129,7 @@ else()
 	)
 endif()
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 	# One needs to have at least one hardware driver present, otherwise
 	# there is no point compiling just the dispatcher.
 	if(PIGLIT_BUILD_DMA_BUF_TESTS)
@@ -160,7 +160,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 		list(APPEND UTIL_GL_LIBS ${WAYLAND_LDFLAGS})
         endif()
 
-endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 
 if(MSVC)
 	list(APPEND UTIL_SOURCES msvc/getopt.c)
